packs/feats/edgewatch-detective-dedication.json

```diff
@@ -26,15 +30,283 @@
         },
         "rules": [
             {
-                "key": "ActiveEffectLike",
-                "mode": "upgrade",
-                "path": "system.skills.society.rank",
-                "value": 1
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.Skill.Acrobatics",
+                        "predicate": [
+                            {
+                                "gte": [
+                                    "skill:society:rank",
+                                    1
+                                ]
+                            },
+                            {
+                                "gte": [
+                                    "skill:thievery:rank",
+                                    1
+                                ]
+                            }
+                        ],
+                        "value": "acrobatics"
+                    },
+                    {
+                        "label": "PF2E.Skill.Arcana",
+                        "predicate": [
+                            {
+                                "gte": [
+                                    "skill:society:rank",
+                                    1
+                                ]
+                            },
+                            {
+                                "gte": [
+                                    "skill:thievery:rank",
+                                    1
+                                ]
+                            }
+                        ],
+                        "value": "arcana"
+                    },
```

Comment:
What is happening here?

We shouldn't do anything with the "if you're already trained pick a skill" thing, we'll figure out some clever way to handle that when we do skill training management